src/generic/listctrl.cpp

```diff
@@ -1601,6 +1616,9 @@ void wxListMainWindow::Init()
     m_anchor  = (size_t)-1;
 
     m_hasCheckBoxes = false;
+
+    m_listRulesAlternateColourOnBlank = false;
+    m_alternateColourOnBlank = wxColour(176, 235, 212, 0xff);;
```

Comment:
We really, really shouldn't hardcode the colour here. It should be based on some system colour and at the very least differ between light and dark modes (see `wxSystemAppearance::IsUsingDarkBackground()`).